src/handbook/company/security/incident-response.md

```diff
@@ -0,0 +1,167 @@
+    are not verified and require further investigation. There is no clear indicator
+    that systems have tangible risk and do not require emergency response. This
+    includes lost/stolen laptop with disk encryption, suspicious emails, outages,
+    strange activity on a laptop, etc.
+ - **P2** - High Severity
+    High severity issues relate to problems where an adversary or active
+    exploitation hasn't been proven yet, and may not have happened, but is likely
+    to happen. This may include lost/stolen laptop without encryption, vulnerabilities
+    with direct risk of exploitation, threats with risk or adversarial persistence
+    on our systems (e.g.: backdoors, malware), malicious access of business
+    data (e.g.: passwords, vulnerability data, payments information).
+ - **P1** - Critical Severity
+    Critical issues relate to actively exploited risks and involve a malicious
+    actor or threats that put any individual at risk of physical harm.
+    Identification of active exploitation is required to meet this severity category.
+
+### Escalation and Internal Reporting
+
+ - **P1** - Critical Severity
+    P1 issues require immediate notification to a member of the [SIRT](#security-incident-response-team-sirt)
+ - **P2** - High Severity
+    An issue should be raised in [`flowforge/security`](https://github.com/flowforge/security){rel="nofollow"}
+    and explicit notification in the #security channel
+ - **P3/P4** - Medium and Low Severity
+    An issue should be raised in [`flowforge/security`](https://github.com/flowforge/security){rel="nofollow"}
+
+### Documentation
+
+All reported security events, incidents, and response activities shall be documented
+and adequately protected in the `flowforge/security` repository.
+
+A root cause analysis (RCA) may be performed on all verified **P1** security incidents.
+It will be reference in the incident issue and reviewed by the SIRT.
+
+## Incident Response Process
+
+For critical issues, the response team will follow an iterative response process.
+
+This covers the following phases:
+
+1. **Investigate** - establish the known facts of the situation.
+2. **Contain** - limit the impact of the situation.
+3. **Eradicate** - remove the immediate cause of the situation.
+4. **Recover** - restore the affected systems and services.
+5. **Remediate** - apply necessary preventative measures to ensure the situation cannot happen again.
+6. **Document** - perform a post-mortem of the situation and a root-cause analysis (RCA). Document to ensure lessons can be learnt.
+
+
+## External Communications and Breach Reporting
+
+In the event of unauthorized access to company or customer systems, networks,
+and/or data, the CEO, with Legal advice, determine what external communications
+are required.
+
+Breaches shall be reported to customers, consumers, data subjects and regulators
+without undue delay and in accordance with all contractual commitments and
+applicable legislation.
```

Comment:
Please add to: https://flowforge.com/handbook/marketing/channels/#content-channels

Further, how are you going to make sure we actually do this? Patch releases don't have blog posts either at this time, while listed in the table linked above